tests/test_channels/test_partial_trace.py

```diff
@@ -34,7 +34,7 @@ def test_partial_trace_sys():
 
     expected_res = np.array([[12, 14], [20, 22]])
 
-    res = partial_trace(test_input_mat, 1)
+    res = partial_trace(test_input_mat, [0])
```

Comment:
Similarly to the other comment about the `partial_transpose`, we should have a test for when the user supplies an `int` argument for `sys`. Something like:

```python
def test_partial_trace_sys_int():
    """Supply `sys` argument as int."""
    test_input_mat = np.arange(1, 17).reshape(4, 4)

    expected_res = np.array([[12, 14], [20, 22]])

    res = partial_trace(test_input_mat, 0)

    bool_mat = np.isclose(expected_res, res)
    np.testing.assert_equal(np.all(bool_mat), True)
```